xcode/routing/routing.xcodeproj/project.pbxproj

```diff
@@ -1069,7 +1069,8 @@
 		675343F01A3F640D00A0A8C3 /* Project object */ = {
 			isa = PBXProject;
 			attributes = {
-				LastUpgradeCheck = 1410;
+				BuildIndependentTargetsInParallel = YES;
+				LastUpgradeCheck = 1510;
```

Comment:
Grep shows that we have this entry for all projects.
Why upgrade only 5 of them?

Reply:
Good point, thanks. Fixed all projects.